docs/contributors/releasing.md

```diff
@@ -32,7 +32,7 @@ between two last release tags. It can be run using scala-cli:
 
 ```
 cs install scala-cli 
-scala-cli ./bin/merged_prs.sc -- <tag1> <tag2> "<github_api_token>"
+scala-cli ./bin/merged_prs.scala -- <tag1> <tag2> "<github_api_token>"
 ```
 
 It will need a [basic github API token](https://github.com/settings/tokens) (don't need any additional scopes) to run, which may be specified via
```

Comment:
I think you should update the info about the `GITHUB_TOKEN` env var.

Reply:
Thanks, fixed!